db/bigtable_eth1.go

```diff
@@ -2475,11 +2475,11 @@ func (bigtable *Bigtable) GetAddressErc721TableData(address string, search strin
 
 	tableData := make([][]interface{}, len(transactions))
 	for i, t := range transactions {
-		from := utils.FormatHash(t.From)
+		from := utils.FormatAddressWithLimits(t.From, "", false, "", 11, 0, false)
 		if fmt.Sprintf("%x", t.From) != address {
 			from = utils.FormatAddressAsLink(t.From, "", false, false)
 		}
-		to := utils.FormatHash(t.To)
+		to := utils.FormatAddressWithLimits(t.To, "", false, "", 11, 0, false)
```

Comment:
This changes the view a bit.
Before the changes FormatHash showed the first and last 2 bytes (so 4 characters)
With the parameter 11 you only show the first and last 3 characters. (11-5 = 6 => 6/2 = 3).
You want 13.

I used /address/0x78b155d7bc284a887f09e028a360a36cc4c4d4c8#erc721Txns to check.

Reply:
tnx, fixed

Reply:
Only "to" was changed, "from" also needs to be done. (again also in the other method too)

Reply:
true that ... fixed